docs/docs/dev_docs/testing/cheat_codes.md

```diff
@@ -0,0 +1,449 @@
+---
+title: Testing Cheat Codes
+---
+
+## Introduction
+
+Most of the time, simply testing your smart contract isn't enough. To manipulate the state of the Aztec blockchain, as well as test for specific reverts and events, the sandbox is shipped with a set of cheatcodes.
+
+Cheatcodes allow you to change the time of the L2 block, load certain state or more easily manipulate L1 instead of having to write dedicated RPC calls to anvil. 
+
+:::info Prerequisites
+If you aren't familiar with [Anvil](https://book.getfoundry.sh/anvil/), we recommend reading up on that since Aztec Sandbox uses Anvil as the local L1 instance.
+:::
+
+### Aims
+
+The guide will cover how to manipulate the state of the:
+- L1 blockchain
+- Aztec network.
+### Why is this useful?
+
+<!-- Contextualise why a dapp developer needs this. What use cases / products / features does this unlock? Any real world examples? -->
```

Comment:
You should probably create an issue for this such that it is not forgotten or finish it as part of this pr.